### ISSUE TITLE
Can anyone make it for J7 Prime? it has the same chipset as in Rpi3

### ISSUE THREAD

**Issue by wolfy9080** [seemoo-lab/nexmon#219]
Ive been looking into this for months. Samsung galaxy J7 prime has bcm43430 by the way the same is present in Raspberry Pi 3, is there is anyway to get monitor mode working?

**Comment by mirzaatifbaig** [comment]
I don't think it will work on your device.And for the binary file make it yourself.Get chroot install git GCC make automaker and other dependencies.Clone the repo(or the folder)make and voila you got the .bin file replace and install the bins and libs using the nexmon app.
